ConsoleAppHomeworkRecurcion/ConsoleAppHomeworkRecurcion/Program.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+
+class ConsoleAppHomeworkRecurcion
+{
+    static void Main(string[] args)
+    {
+        //First task is so hard for me, sorry...
+        //2 task:
+        Console.WriteLine(Pow(2,2));
+        //3 task:
+        Foo(10);
+    }
+    static int Pow(int x, int y, int i = 1)
```

Comment:
it's not critical, but you can do it with single parameter Y and instead of increasing I you can decrease Y